forms-flow-web/src/components/Form/List.js

```diff
@@ -278,6 +278,8 @@ const List = React.memo((props) => {
     };
     addUsers(payload).catch((error) => console.error("error", error));
```

Comment:
in future need to change function name 
addUsers -> addDesigners

Reply:
Noted